### ISSUE TITLE
rewrite

### ISSUE THREAD

**Issue by gilgoldzweig** [pchmn/MaterialChipsInput#6]
after looking at your code and edit it I found lots of bad code and bad performances,
the project could be much better, faster.
If you need help contact me in private

**Comment by pchmn** [comment]
Yeah it's a first version and I can't work on the project 24/7, so this is a bit messy and not fully optimized. 
It worked so I published the library, but the next step is to optimize the code.
I'll contact you

**Comment by searchy2** [comment]
@gilgoldzweig seems a bit harsh. I've been watching this repo for a while and @pchmn is one of the best at keeping on top of new issues and merging PRs. Chips are something that Google should probably have released a library for but instead, good people like this repo's author have to step up instead.

**Comment by pchmn** [comment]
Thanks for your support man !
